src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Monitor your Cordova application
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Cordova
+  - Get started
+metaDescription: "New Relic's Cordova agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started/new-relic-cordova-overview
+  - /docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started/introduction-new-relic-cordova 
+---
+
+import mobileHybridSample from 'images/mobile_screenshot-full_hybrid-summary.png'
+
+import mobileQueryExample from 'images/mobile_screenshot-full_query-example.png'
+
+The New Relic Cordova agent monitors native applications (iOS, Android) built on the [Apache Cordova framework](https://cordova.apache.org/docs/en/latest/guide/overview/index.html). Written in JavaScript, the Cordova agent also includes all New Relic mobile monitoring features for native apps.
+
+## Features
+
+Once installed, the Cordova agent can:
+
+* Detect the platforms added to your Cordova application and apply the most recent release of the relevant New Relic [Android](/docs/releases/android) or [iOS](/docs/releases/ios) agent
+* Add post-build scripts for uploading iOS symbolication files
+* Upload Android Proguard mapping files
+* Automatically instrument mobile applications built via Cordova
+* Capture JavaScript errors
+* Generate distributed traces
+* Track promise rejection (unhandled exceptiona in JavaScript)
+* Track warnings and errors with console logs
+* Capture interactions and their sequences
+* Track user sessions
+
+## Get started with Cordova monitoring
+
+### Step 1: Check the requirements [#prerequisites]
+
+Before installing, make sure you've met these prerequisites for adding the Android or iOS platform to your Cordova project:
+
+* Cordova 7.x or higher
+* Node.js 6.0 or higher
+* Cordova CLI tools
+* An [Android](https://cordova.apache.org/docs/en/latest/guide/platforms/android) or [iOS](https://cordova.apache.org/docs/en/latest/guide/platforms/ios/index.html) Cordova platform for your project
+* Mobile [application tokens](/docs/mobile-monitoring/new-relic-mobile/maintenance/viewing-your-application-token)
+
+### Step 2: Get your application token [#step-2]
+If you’re new to New Relic, [create an account](https://newrelic.com/signup) - it's free!
+
+First, you'll need to get your application token, which will be used in [Step 3](#install) to connect your Cordova app with New Relic:
```

Comment:
I'd try to avoid the passive form here. Maybe this one or something similar?
First, you need to get your application token to use later in Step 3.